kubetest/main.go

```diff
@@ -495,7 +492,7 @@ func migrateGcpEnvAndOptions(o *options) error {
 			name:   "--gcp-network",
 		},
 		{
-			env:    nodeImage,
+			env:    "KUBE_NODE_OS_DISTRIBUTION",
 			option: &o.gcpNodeImage,
```

Comment:
Not that I can think of. I believe gcloud will take `gci` as well, but I'll confirm in just a bit if I get a change.